h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -42,36 +47,80 @@ function Label({
   );
 }
 
-export default function CreateGroupForm() {
-  const nameId = useId();
-  const descriptionId = useId();
+function NameField() {
```

Comment:
Yeah that's much better, especially because a future commit is going to have to gather the values of the two fields together in one place to send the API request, so I'm going to need them as state then anyway. Done: https://github.com/hypothesis/h/pull/8804/commits/b24f4bb2beafac8fb8dcf4854c5ca5af26fbde35